pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -17,10 +17,19 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/knative/pkg/apis"
 	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	"github.com/knative/pkg/webhook"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Check that Task may be validated and defaulted.
+var _ apis.Validatable = (*TaskRun)(nil)
+var _ apis.Defaultable = (*TaskRun)(nil)
+
+// Assert that Task implements the GenericCRD interface.
```

Comment:
i think the comments here should say `TaskRun` (or just "it" so we dont have to update it each time we copy it XD)